CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contribute to TheRock
+
+We are enthusiastic about contributions to our code and documentation. For contribution guidelines
+to other parts of ROCM outside of **TheRock**, please see
+[CONTRIBUTING.md](https://github.com/ROCm/ROCm/blob/develop/CONTRIBUTING.md).
+
+## Development workflow
+
+**TheRock** is focused on making it easy for developers to contribute. In order to facilitate this,
+the source of truth for all issue tracking, project planning and code contributions are in GitHub and
+we leverage an open source stack for all development tools and infrastructure so that it they can be
+easily leveraged in any fork.
+
+The project is still in its early days, so please feel free to file bugs where documentation or
+functionality is lacking or even better volunteer to help contribute to help close these gaps!
+
+### Issue tracking
+
+Before filing a new issue, search the
+[existing issues](https://github.com/ROCm/therock/issues) to make sure your issue isn't
+already listed.
+
+General issue guidelines:
+
+- Use your best judgement for issue creation. If your issue is already listed, upvote the issue and
+  comment or post to provide additional details, such as how you reproduced this issue.
+- If you're not sure if your issue is the same, err on the side of caution and file your issue.
+  You can add a comment to include the issue number (and link) for the similar issue. If we evaluate
+  your issue as being the same as the existing issue, we'll close the duplicate.
+- When filing an issue, be sure to provide as much information as possible, including script output so
+  we can collect information about your configuration. This helps reduce the time required to
+  reproduce your issue.
+- Check your issue regularly, as we may require additional information to successfully reproduce the
+  issue.
+
+### Governance
+
+This is currently covered by the
+[ROCM project Governance](https://github.com/ROCm/ROCm/blob/develop/GOVERNANCE.md).
+
+### Pull requests
+
+When you create a pull request, you should target the *main* branch.
+
+- Identify the issue you want to fix
+- Target the main branch
```

Comment:
Unfortunate that we even need to specify which branch to target... but makes sense given current contributing practices in other ROCm organization repositories.